src/compiler/types.ts

```diff
@@ -5846,6 +5909,45 @@ export const enum SymbolFlags {
     LateBindingContainer = Class | Interface | TypeLiteral | ObjectLiteral | Function,
 }
 
+export interface SymbolDisplayPart {

```

Comment:
Sort of a bummer to have to have these here, but I don't see a way around that. Why oh why did we introduce the methods... 😄